MozillaFoundation/foundation.mozilla.org
# Add "wide" option to Datawrapper

# Description

This PR updates the `datawrapper` block to a `StuctBlock` and adds an additional `Wide` option checkbox to make the `datawrapper` expand beyond the regular content width on large screens.

Similar to the [Add `wide` option to Table PR](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12843), except the default here is that it should not be wide because it currently isn't on production.

Related PRs/issues: TP1-143 / #10331 / #10163

# To Test
1) View [an example page on production](http://foundation.mozilla.org/en/privacynotincluded/articles/our-2023-nice-list-we-see-you-good-at-privacy-connected-products/) with a datawrapper
2) Note the datawrapper is the same width as the content, and that there is no `Wide` checkbox option on the edit screen.
3) Checkout this branch locally
4) Pull the database from live / stage
5) View the [example page locally
](en/privacynotincluded/articles/our-2023-nice-list-we-see-you-good-at-privacy-connected-products/)
6) Verify the datawrapper is still there and has the same width as the content.
7) Verify that on the edit screen that there is now a `Wide` checkbox for the datawrapper.
8) Check the `Wide` checkbox for a datawrapper and save the page
9) Verify the datawrapper now expands beyond the content width (but remains same width on mobile).

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1220)
